x-pack/plugins/reporting/server/lib/export_types_registry.ts

```diff
@@ -86,4 +86,8 @@ export class ExportTypesRegistry {
 
     return result;
   }
+
+  keys() {
+    return this._map.keys();
+  }
```

Comment:
We should try use existing parts of the ExportTypesRegistry for testing, instead of adding a new part for a test